test/test_framework.py

```diff
@@ -588,9 +596,11 @@ class MyEventsA(EventsBase):
         class MyEventsB(EventsBase):
             pass
 
-        class MyNotifier(Object):
-            on_a = MyEventsA()
-            on_b = MyEventsB()
```

Comment:
This test was verifying that we're able to do this, specifically. By changing it we're now testing what?  A better change would be making sure that it does work despite caching, instead of invalidating the test. It would be okay to manually change the handle_kind in MyEventsA and MyEventsB, if that's what it takes.